tsl/test/shared/expected/ordered_append_join-15.out

```diff
@@ -1854,20 +1854,25 @@ ORDER BY time DESC
 LIMIT 2;
 QUERY PLAN
  Limit (actual rows=2 loops=1)
-   ->  Sort (actual rows=2 loops=1)
-         Sort Key: _hyper_X_X_chunk."time" DESC
-         Sort Method: top-N heapsort 
-         ->  Nested Loop (actual rows=136740 loops=1)
-               ->  Append (actual rows=68370 loops=1)
-                     ->  Custom Scan (DecompressChunk) on _hyper_X_X_chunk (actual rows=25190 loops=1)
-                           ->  Seq Scan on compress_hyper_X_X_chunk (actual rows=30 loops=1)
+   ->  Nested Loop (actual rows=2 loops=1)
+         ->  Custom Scan (ChunkAppend) on metrics_compressed (actual rows=1 loops=1)
+               Order: metrics_compressed."time" DESC
+               ->  Sort (actual rows=1 loops=1)
+                     Sort Key: _hyper_X_X_chunk."time" DESC
+                     Sort Method: quicksort 
                      ->  Custom Scan (DecompressChunk) on _hyper_X_X_chunk (actual rows=25190 loops=1)
                            ->  Seq Scan on compress_hyper_X_X_chunk (actual rows=30 loops=1)
```

Comment:
Shouldnt the Sort be pushed into the DecompressChunk node and either produce ordered output or use batch merge sort

Reply:
Yes, it is possible to apply this optimization. However, to prevent flaky tests, it is disabled for these tests (see line 1843).